pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -720,6 +732,53 @@ func (c *Cluster) getPVCMetadataInitContainerActivate(mountPath string, osdProps
 	}
 }
 
+func (c *Cluster) getPVCWaldataInitContainer(mountPath string, osdProps osdProperties) v1.Container {
+	return v1.Container{
+		Name:  blockPVCWaldataMapperInitContainer,
+		Image: c.cephVersion.Image,
+		Command: []string{
+			"cp",
+		},
+		Args: []string{"-a", fmt.Sprintf("/%s", osdProps.waldataPVC.ClaimName), fmt.Sprintf("/srv2/%s", osdProps.waldataPVC.ClaimName)},
```

Comment:
what about using a dir called `/srvwal`?

Reply:
/srvwal will have problems when check the prefix of /srv (metadata) and /srvwal(wal), I change the dir to /wal directly.